project/Driver.scala

```diff
@@ -150,6 +152,7 @@ final class Driver(core: Project) {
       }
     }
     .dependsOn(core)
+    .dependsOn(actorModule % "compile->compile;test->test")
```

Comment:
As far as I see, there is no test in this "binding" module, so the scopes there are not required

Reply:
No tests, but there is code for the test scope in there: https://github.com/ReactiveMongo/ReactiveMongo/pull/1245/files#diff-a2cdc7f0cb0342178b80349f37ccab46ca61781e5cc29e56e5b156099620eb67